GetStream/stream-chat-swift
# [CIS-473] SuggestionViewController UI

# What this PR does
- Implements UI for SuggestionsViewController with collectionView and its cells.
# How to test this
- This is just UI, co you can go to any chat in the demo app and try to display suggestions via `/`, do not expect any logic right now, that is WIP. 

#UI preview:
<img width="428" alt="Mentions" src="https://user-images.githubusercontent.com/17381941/101939435-42406c80-3be5-11eb-9b07-fe9060594835.png">
<img width="422" alt="Commands" src="https://user-images.githubusercontent.com/17381941/101939439-44a2c680-3be5-11eb-9ca6-f98527a80e35.png">


Please keep in mind that the logic of presenting this viewController is to be defined in next steps, it should probably be presented on `ChatChannelVC` or separate window because the inputWindow doesn't work for us. Also the max 4 items height of the suggestions will follow in the logic commit. 

Thanks @evsaev for help with debugging the collectionViewLayout!
